src/component/shortcuts.js

```diff
@@ -77,8 +77,13 @@ export default class extends React.Component {
       let isInputLikeElement = element.tagName === 'INPUT' ||
         element.tagName === 'SELECT' || element.tagName === 'TEXTAREA' ||
           (element.contentEditable && element.contentEditable === 'true')
-      let isReturnString = event.key && event.key.length === 1
-
+        var isReturnString;
```

Comment:
Please, respect project coding style: no semi-colons and `let` instead of `var`.